testdata/ecosystem/pipelines/nodejs.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: nodejs-ex-git
```

Comment:
pipeline name is not the same as file name

Reply:
Done